cg/store/models.py

```diff
@@ -1075,6 +1075,37 @@ class IlluminaSequencingRun(InstrumentRun):
     __mapper_args__ = {"polymorphic_identity": DeviceType.ILLUMINA}
 
 
+class PacBioSequencingRun(InstrumentRun):
+    __tablename__ = "pacbio_sequencing_run"
+
+    id: Mapped[int] = mapped_column(ForeignKey("instrument_run.id"), primary_key=True)
+    well: Mapped[Str32 | None]
+    plate: Mapped[int | None]
+    movie_time_hours: Mapped[int | None]
+    hifi_reads: Mapped[BigInt | None]
+    hifi_yield: Mapped[BigInt | None]
+    hifi_mean_read_length: Mapped[BigInt | None]
+    hifi_median_read_quality: Mapped[Str32 | None]
+    percent_reads_over_q30: Mapped[Num_6_2 | None]
+    productive_zmw: Mapped[BigInt | None]
+    p0_zmw: Mapped[BigInt | None]
+    p1_zmw: Mapped[BigInt | None]
+    p2_zmw: Mapped[BigInt | None]
+    percentage_p0: Mapped[Num_6_2 | None]
+    percentage_p1: Mapped[Num_6_2 | None]
+    percentage_p2: Mapped[Num_6_2 | None]
```

Comment:
Should we keep only percentages? or is the number of ZMWs important? SMRT link only shows percentages but the data we will fetch will give us number of ZMW

Reply:
zmw no idea what that abbreviation means.